src/Cli/Abstractions/Azure.Functions.Cli.Abstractions.csproj

```diff
@@ -2,6 +2,11 @@
 
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+	<OutputType>Library</OutputType>
+	<ImplicitUsings>enable</ImplicitUsings>
```

Comment:
nit: fix indentations

You should just need `<OutputType>Library</OutputType>`. nullable nad implicit using should already be set in the common props (feel free to add these if you see warnings though). The name properties will automatically be set to what the csproj file name is so you can remove those